src/collectors/loggly/loggly.py

```diff
@@ -0,0 +1,268 @@
+# coding=utf-8
+
+"""
+Collect numeric value of a field from loggly search results.
+
+This collector will retrieve the values of a configured field from loggly API
+and publish each one on the log's specified timestamp.
+
+The collector will limit the query to a timeframe equal to the collector's
+duration back from now.
+
+#### Dependencies
+
+    * requests
+    * futures
+
+Example config file LogglyCollector.conf
+
+```
+enabled = true
+interval = 60
+
+[ metric ]
+[[ test.metric ]]
+subdomain = example
```

Comment:
Does it make sense to keep the subdomain, username and password as required for each metric? Shouldn't this be a global value for the collector instance? I'm not too familiar with loggly, so don't know if this makes sense. Remember that if multiple passwords are needed, a user could always instantiate another instance of the collector?